roles/agnosticv/templates/catalog_item.yml.j2

```diff
@@ -10,6 +10,7 @@ metadata:
     babylon.gpte.redhat.com/displayName: {{ _catalog.display_name | default(_name) | to_json }}
     babylon.gpte.redhat.com/icon: {{ _icon | to_json if _icon is string else _icon | to_json | to_json }}
     babylon.gpte.redhat.com/keywords: {{ _catalog.keywords | default([]) | join(',') | to_json }}
+    babylon.gpte.redhat.com/last_update: {{ _last_update | to_json }}
```

Comment:
Because `_last_update` is an object, we need to double encode `| to_json | to_json` to get a valid json string that contains json data.